modules/services/way-displays.nix

```diff
@@ -14,22 +14,40 @@ let
     mkEnableOption
     mkIf
     mkOption
+    optionalAttrs
+    optionalString
     types
     ;
   mergeSets = sets: lists.fold attrsets.recursiveUpdate { } sets;
   cfg = config.services.way-displays;
+  serviceName = "way-displays" + optionalString cfg.statefulConfiguration "@";
   yaml = pkgs.formats.yaml { };
 in
 {
-  meta.maintainers = [ maintainers.jolars ];
+  meta.maintainers = [
+    maintainers.jolars
+    maintainers.mightyiam
+  ];
 
   options.services.way-displays = {
     enable = mkEnableOption "way-displays";
 
     package = lib.mkPackageOption pkgs "way-displays" { };
 
+    statefulConfiguration = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        A configuration file is not generated.
+        Allows way-displays to write its own configuration.
+
+        The service is _not_ started automatically.
+        It can be started by running `systemctl --user start "way-displays@$XDG_VTNR".service`.
```

Comment:
I'm not sure I understand. Why don't we want the service started?

Reply:
Outdated